src/kt_variant.h

```diff
@@ -3,6 +3,10 @@
 #include "wire/wire.pb.h"
 #include "core/variant.h"
 
+// must match the value order of godot_variant_type
+static void(*TO_KT_VARIANT_FROM[27 /* Variant::Type count */])(wire::Value&, const Variant&);
```

Comment:
does it need to be in the header? I mean clients of the class shouldn't really know about this arrays.

Reply:
yep that's true, let me change this.